CHANGELOG.md

```diff
@@ -136,6 +136,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 * Ensuring  pod install/update can run on Windows.  
   [David Airapetyan](https://github.com/davidair)
   [#8055](https://github.com/CocoaPods/CocoaPods/issues/8055)
+* Copies the bcsymbolmap files of a vendored framework  
```

Comment:
nit move this to the top in `enhancements` list of master. Its because 1.6.0.rc.2 was released today.